modules/ROOT/pages/standalone-mule-setup.adoc

```diff
@@ -15,8 +15,8 @@ Anypoint Visualizer supports the following Mule versions for standalone Mule dep
 == Install Agent
 
 To support Anypoint Visualizer for standalone Mule deployments, install the xref:monitoring::am-installing.adoc[Anypoint Monitoring agent].
-Anypoint Visualizer also requires an additional header injection for standalone Mule deployments.
-However, due to a potential performance impact for Mule 4, header injection is disabled by default. 
+Anypoint Visualizer requires an additional header injection for standalone Mule deployments so that the deployments display the correct connections.
```

Comment:
```suggestion
Anypoint Visualizer requires header injection for standalone Mule deployments so that the deployments display the correct connections.
```